website/docs/reference/warehouse-profiles/materialize-profile.md

```diff
@@ -43,20 +57,30 @@ dbt-materialize:
 
 </File>
 
+To test the connection to Materialize, run:
+
+```
+dbt debug
+```
+
+If the output reads "All checks passed!", you’re good to go! Check the [dbt and Materialize guide](https://materialize.com/docs/guides/dbt/) to learn more and get started.
+
 ## Supported Features
 
 ### Materializations
 
+Because Materialize is optimized for transformations on streaming data and the core of dbt is built around batch, the `dbt-materialize` adapter implements a few custom materialization types:
```

Comment:
is there a link from the profile info page to the config page? if not, this'd be a great place to add it!

Reply:
There's a link in the indexes entry, but that's it! Right now, the configurations page doesn't really provide more than index-specific information, so do you think it'd make sense to link it elsewhere? 

I was actually wondering if this section as a whole would actually be a better fit in the configurations page, rather than the profile. WDYT?

Reply:
Every other documented database seems to do its own thing, so I wasn't too sure!